stl/inc/ranges

```diff
@@ -8440,6 +8446,9 @@ namespace ranges {
     _EXPORT_STD template <class _Container, class... _Types>
         requires (!view<_Container>)
     _NODISCARD constexpr auto to(_Types&&... _Args) {
+        static_assert(!is_const_v<_Container>, "C must not be const. (N4928 [range.utility.conv.adaptors])");
+        static_assert(!is_volatile_v<_Container>, "C must not be volatile. (N4928 [range.utility.conv.adaptors])");
+        static_assert(is_class_v<_Container>, "C must be a class type. (N4928 [range.utility.conv.to])");

```

Comment:
\[range.utility.conv.to\] was copy-pasted here; it should be \[range.utility.conv.adaptors\] like the first two lines.